packages/pangea-node-sdk/src/types.ts

```diff
@@ -223,26 +223,21 @@ export namespace Audit {
   }
 
   export enum DownloadFormat {
-    /**
-     * JSON.
-     */
+    /** JSON. */
     JSON = "json",
 
-    /**
-     * CSV.
-     */
+    /** CSV. */
     CSV = "csv",
   }
 
   export interface DownloadRequest {
-    /**
-     * ID returned by the search API.
-     */
-    result_id: string;
+    /** ID returned by the search API. */
+    request_id?: string;
 
-    /**
-     * Format for the records.
-     */
+    /** ID returned by the search API. */
```

Comment:
Same comment than in python https://github.com/pangeacyber/pangea-python/pull/337#discussion_r1588260992

Reply:
Corrected now.